src/views/partials/header.ejs

```diff
@@ -1,16 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title><%- title %></title>
-  <% if (typeof styles !== 'undefined') { %>
-    <% if (styles.length > 0) { %>
-            <link rel="stylesheet" href="css/<%= styles %>.css">
-    <% } %>
-<% } %>
-</head>
-<body>
-  <!--Todo: Navbar-->
-  <main>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <!-- <meta http-equiv="X-UA-Compatible" content="IE=edge" /> -->
```

Comment:
Why are these 2 boilerplates commented out? Why are the if statements on line 8-10 commented out? The first if command should have index.css and then you can remove the style from the controllers I believe